pom.xml

```diff
@@ -113,7 +113,7 @@
         <cs.junit.dataprovider.version>1.13.1</cs.junit.dataprovider.version>
         <cs.junit.jupiter.version>5.9.1</cs.junit.jupiter.version>
         <cs.guava-testlib.version>18.0</cs.guava-testlib.version>
-        <cs.mockito.version>3.2.4</cs.mockito.version>
```

Comment:
we may also need the following change, so mockstatic() can be used with Mockito itself.

-      <artifactId>mockito-core</artifactId>
+      <artifactId>mockito-inline</artifactId>

Let's first run tests and see nothing is broken

Reply:
@harikrishna-patnala tests have been run in the github acrions. Do you need to see anything else?

Reply:
@harikrishna-patnala Let me make this change. Then we can incrementally remove PowerMockito.